src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/db/UserData.kt

```diff
@@ -0,0 +1,39 @@
+package org.vaccineimpact.orderlyweb.db
+
+interface UserData
+{
+    fun addGithubUser(username: String, email: String)
+}
+
+class OrderlyUserData : UserData
+{
+    override fun addGithubUser(username: String, email: String)
+    {
+        addUser(username, email, "github")
+    }
+
+    private fun addUser(username: String, email: String, source: String)
```

Comment:
I think it's nice to have a name to show to people. Github users all have a username and optionally have a display name; we could store both, or we could store a single property "name" which we populate from the display name if available, otherwise the username. Maybe for completeness actually it's nice to have both.

Reply:
Have also added a `lastLoggedIn` field while I was at it.